System/Console/ANSI/Windows/Emulator.hs

```diff
@@ -294,21 +293,31 @@ applyANSISGRToAttribute def sgr attribute = case sgr of
     (attribute .&. (complement bACKGROUND_INTENSITY))
   SetColor Background Vivid color -> applyBackgroundANSIColorToAttribute color
     (attribute .|. bACKGROUND_INTENSITY)
+  SetColor8 layer color -> applyANSISGRToAttribute def (SetColor8Code layer $ color8ToCode color) attribute
+  SetColor8Code layer code -> let intensity = fst res
```

Comment:
I wondered if:
~~~~
let (intensity, col) = approximateColor8Code code
in  applyANSISGRToAttribute def (SetColor layer intensity col) attribute
~~~~
would be 'clearer' code?

Reply:
oh yes, indeed